.env.production

```diff
@@ -2,3 +2,4 @@ VITE_PROVIDER_URL=wss://hydradx-rpc.play.hydration.cloud
 VITE_DOMAIN_URL=https://app.basilisk.cloud
 VITE_INDEXER_URL=https://basilisk-devnet-explorer.play.hydration.cloud/graphql
 VITE_FF_POOLS_ENABLED=false
+VITE_FF_XCM_ENABLED=true
```

Comment:
is this correct? hidden for rococo, enabled for prod?

Reply:
yes ..